lizzy_client/lizzy.py

```diff
@@ -77,8 +77,18 @@ def get_stacks(self) -> list:
         request.raise_for_status()
         return request.json()
 
-    def new_stack(self, image_version: str, keep_stacks: int, new_traffic: int, senza_yaml_path: str,
-                  application_version: Optional[str], disable_rollback: bool, parameters: List[str]) -> str:
+    def new_stack(self,
+                  image_version: str,
+                  keep_stacks: int,
+                  new_traffic: int,
+                  senza_yaml_path: str,
+                  stack_version: Optional[str],
```

Comment:
Ok. The only concern I am having with this code is the number of parameters and the code readability of methods like that with a lot non-named arguments. Because the developer or reviewer have quite some trouble to understand calls like:

```python
lizzy.new_stack('10', 2, 42, "./s.yaml", "7", "420", True, ['abc', 'def'])
```

Questions that came to my mind at first glance on method calls like that:
- Is it passing the correct arguments where it should be?
- Is the number of arguments matching the number of arguments the method actually expects?
- What is going to be the value of the variable `foobar` in this method call?

Of course, some of those questions are easily handled by the tests but still keeps me wondering what exactly is going on and only possible to be clearly visible under a debugger.